app/src/main/java/atorch/statspuzzles/SolvePuzzle.java

```diff
@@ -324,6 +331,31 @@ private void showHint() {
             ((TextView) alert.findViewById(android.R.id.message)).setMovementMethod(LinkMovementMethod.getInstance());
         }
 
+        private void onGeminiHint() {
+            String puzzleText = res.getPuzzle(level, puzzleIndex);
+            String prompt = "Can you give me a hint for this puzzle, without giving away the answer? Here's the puzzle: " + puzzleText;
```

Comment:
Could you add this prompt to the translated strings? It's more consistent for the German translation if the prompt is in German as well, just to make sure that the answers come back in German.

Reply:
Good call, thank you for the comment